test/e2e/storage/empty_dir_wrapper.go

```diff
@@ -387,14 +386,8 @@ func testNoWrappedVolumeRace(f *framework.Framework, volumes []v1.Volume, volume
 				Spec: v1.PodSpec{
 					Containers: []v1.Container{
 						{
-							Name:    "test-container",
-							Image:   imageutils.GetE2EImage(imageutils.BusyBox),
-							Command: []string{"sleep", "10000"},
-							Resources: v1.ResourceRequirements{
-								Requests: v1.ResourceList{
-									v1.ResourceCPU: resource.MustParse("10m"),
-								},
-							},
+							Name:         "test-container",
+							Image:        imageutils.GetE2EImage(imageutils.Pause),
```

Comment:
My only q would be whether k8s has special behavior around the `pause` image, that would somehow make this a non-representative test? My guess is that this should work fine, but just wanted to double check :)

Reply:
I think there should be no difference in behaviour, just a nicer way of `sleep`'ing :)